source/security/private-endpoints.txt

```diff
@@ -100,111 +93,82 @@ connection using either the App Services UI or the Admin API.
 
          .. step:: Create an Interface Endpoint in Your VPC
 
-            To use a Private Endpoint, you must first create a VPC
-            interface endpoint in AWS. You can do this using the
-            :aws:`create-vpc-endpoint
-            </cli/latest/reference/ec2/create-vpc-endpoint>` command in the
-            AWS CLI. The UI will walk you through creating this command.
-
-            a. In the left navigation, click :guilabel:`App Settings`.
-
-            #. Click the :guilabel:`Private Endpoints` tab.
-
-            #. Click the :guilabel:`Add Private Endpoint` button.
-
-            #. Enter your **VPC ID** in the input box.
-
-            #. Enter your **Subnet IDs** as a comma-separated list in the input box.
-
-            Once you've entered this information, the fully constructed
-            AWS CLI command is displayed in the UI. Copy the command and
-            run it in your terminal. It should resemble the following:
-
+            a. In the App Services UI, navigate to :guilabel:`App Settings` and
+               then click the :guilabel:`Private Endpoints` tab.
+            #. Click :guilabel:`Add Private Endpoint`, then click
+               :guilabel:`General`.
+            #. Add your VPC ID and Subnet IDs (comma-separated list), then click
+               :guilabel:`Next`.
+            #. In your terminal, create a VPC Endpoint with the command generated
```

Comment:
feels like there's a step missing. when does the generated command display for me to copy? in the modal _after_ you click Next?

Reply:
Yeah, you're right. The UI automatically generates it when you add the fields. You don't need to do anything. But I didn't explicitly say users should copy that generated endpoint. I've revised to make the process a bit clearer.